cmd/kexpand/README.md

```diff
@@ -0,0 +1,42 @@
+# kexpand
+
+_TODO: flesh out this placeholder documentation_
+ 
+`kexpand` is a kubernetes cluster configuration utility,
+a prototype of ideas discussed in [this doc][DAM].
+
+It accepts one or more file system path arguments,
+reads the content thereof, and emits kubernetes
+[resource] yaml to stdout, suitable for piping 
+into [kubectl] for [declarative] application to a
+kubernetes cluster.
+
+For example, if your current working directory contained
+> ```
+> mycluster/
+>   Kube-Manifest.xml
```

Comment:
You don't actually mean XML, I hope. :-)

Reply:
lol, sorry fixed